Fix significand docstring

0 opened the pull request:
It previously implied that the result cannot be negative for non-zero finite numbers.

kshyatt commented on base/math.jl:
```diff
@@ -812,17 +812,21 @@ end
+Extract the significand (a.k.a. mantissa) of a floating-point number. If `x` is
+a non-zero finite number, then the result will be a number of the same type and
+sign as `x`, and whose absolute value is on the interval ``[1,2)``. Otherwise
+`x` is returned.
```
While you're here, would you mind adding a doctest for this case? `Inf` or `NaN` would be great.

0 replied:
Added.

0 committed:
Fix significand docstring

It previously implied that the result cannot be negative for non-zero
finite numbers.
